### PR TITLE
create model tariff

### PR DESCRIPTION
**Pull request by romigos**

### CODE REVIEW

File: db/migrate/20181219095843_create_tariffs.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+class CreateTariffs < ActiveRecord::Migration[5.2]
+  def change
+    create_table :tariffs do |t|
+      t.string :name
+      t.float :price
```
Comment (saviokmua) [inline db/migrate/20181219095843_create_tariffs.rb]: we need use integer for money

**Review by saviokmua — CHANGES_REQUESTED** [review]

**Comment by saviokmua** [comment]
@romigos you need to add in comment link  to trello task on this pull request

**Review by saviokmua — APPROVED** [review]

File: app/views/tariffs/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<br>
+<br>
+<br>
+<h2>Редагування тарифу</h2>
+<%= form_for :tariff, url: tariffs_path do |f| %>
+  <p>
+    Назва тарифу<br>
+    <%= f.text_field(:name, {:class => 'form-control'}) %>
```
Comment (saviokmua) [inline app/views/tariffs/edit.html.erb]: @romigos please use new syntaxes
`:class => 'form-control'` -> `class: 'form-control'`

File: app/views/tariffs/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<br>
+<br>
+<h1>Всі тарифи </h1>
+<% @tariff.each do |tariff| %>
+  <div class="alert alert-light">
+    <H4><%=tariff.name %></H4>
```
Comment (saviokmua) [inline app/views/tariffs/index.html.erb]: @romigos please use styles guide formatted
`<H4><%=tariff.name %></H4>` -> `<H4><%= tariff.name %></H4>`

File: app/controllers/tariffs_controller.rb [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    @tariff = Tariff.find(params[:id])
+  end
+  def create
+
+    end
+  def destroy
+    @tariff = Tariff.find(params[:id])
+    @tariff.destroy
+    redirect_to tariffs_path
+    end
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb:33]: @romigos you need to format code

File: app/controllers/tariffs_controller.rb [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    @tariff = Tariff.all
+  end
+  def new
+    @tariff = Tariff.new
+  end
+  def show
+    @tariff = Tariff.find(params[:id])
+  end
+  def edit
+    @tariff = Tariff.find(params[:id])
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb]: @romigos duplicate code. You need create some method like 
```
def bundle_tariff
  @tariff = Tariff.find(params[:id])
end

```

Comment (saviokmua) [reply app/controllers/tariffs_controller.rb]: @romigos and use it
```
def new
  bundle_tariff
end
```

**Comment by romigos** [comment]
https://github.com/saviokmua/print-km-ua/tree/12-tariff

File: db/migrate/20181219095843_create_tariffs.rb (hunk 1 above)
Comment (romigos) [reply db/migrate/20181219095843_create_tariffs.rb]: fixed

File: app/views/tariffs/edit.html.erb (hunk 2 above)
Comment (romigos) [reply app/views/tariffs/edit.html.erb]: fixed

File: app/views/tariffs/index.html.erb (hunk 3 above)
Comment (romigos) [reply app/views/tariffs/index.html.erb]: fixed

File: app/controllers/tariffs_controller.rb (hunk 4 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb:33]: fixed

File: app/controllers/tariffs_controller.rb (hunk 5 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb]: fixed

File: app/controllers/tariffs_controller.rb [hunk 6]
```diff
@@ -1,44 +1,41 @@
...
-
     if @tariff.save
       redirect_to @tariff
     else render 'new'
     end
   end
-private def  tariff_params
+  private
+    def tariff_params
     params.require(:tariff).permit(:name, :price)
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb:43]: please use rubocop format
`rubocop -a /path/to/your/project`

File: app/views/tariffs/index.html.erb [hunk 7]
```diff
@@ -3,8 +3,8 @@
 <h1>Всі тарифи </h1>
 <% @tariff.each do |tariff| %>
   <div class="alert alert-light">
-    <H4><%=tariff.name %></H4>
-    <H4><%=tariff.price %></H4>
+    <H4><%= tariff.name %></H4>
+    <H4><%= tariff.price %></H4>
     <%= link_to "Додати тариф", new_tariff_path(@tariff),:class => 'btn btn-warning' %>
```
Comment (saviokmua) [inline app/views/tariffs/index.html.erb]: @romigos please use new syntax

File: app/views/tariffs/index.html.erb [hunk 8]
```diff
@@ -3,8 +3,8 @@
...
     <%= link_to "Редактувати", edit_tariff_path(@tariff),:class => 'btn btn-warning' %>
     <%= link_to "Видалити ", tariff_path(@tariff), method: :delete, data: {confirm: "Чи ви дійсно хочете видалити тариф?"}, :class => 'btn btn-danger' %>
```
Comment (saviokmua) [inline app/views/tariffs/index.html.erb]: @romigos and here too

File: app/controllers/tariffs_controller.rb [hunk 9]
```diff
@@ -1,44 +1,41 @@
 class TariffsController < ApplicationController
+  def bundle_tariff
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb]: @romigos please move this method to **private** section

File: app/controllers/tariffs_controller.rb [hunk 10]
```diff
@@ -1,44 +1,41 @@
...
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
   end
-
   def update
-    @tariff = Tariff.find(params[:id])
+    bundle_tariff
     if @tariff.update(tariff_params)
       redirect_to @tariff
     else render 'edit'
```
Comment (saviokmua) [inline app/controllers/tariffs_controller.rb]: @romigos please use formatting

Comment (saviokmua) [reply app/controllers/tariffs_controller.rb]:
```
if @tariff.update(tariff_params)
  redirect_to @tariff
else
  render 'edit'
end
```

File: app/views/tariffs/index.html.erb (hunk 7 above)
Comment (romigos) [reply app/views/tariffs/index.html.erb]: fixed

File: app/views/tariffs/index.html.erb (hunk 8 above)
Comment (romigos) [reply app/views/tariffs/index.html.erb]: fixed

File: app/controllers/tariffs_controller.rb (hunk 10 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb]: fixed

File: app/controllers/tariffs_controller.rb (hunk 9 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb]: fixed but don't know is it correct

File: app/controllers/tariffs_controller.rb (hunk 6 above)
Comment (romigos) [reply app/controllers/tariffs_controller.rb:43]: i did not undestend what i have to do in this comment

Comment (saviokmua) [reply app/controllers/tariffs_controller.rb:43]: @romigos  https://github.com/rubocop-hq/rubocop

File: app/views/tariffs/edit.html.erb [hunk 11]
```diff
@@ -0,0 +1,18 @@
+<br>
+<br>
+<br>
+<h2>Редагування тарифу</h2>
+<%= form_for :tariff, url: tariffs_path do |f| %>
```
Comment (saviokmua) [inline app/views/tariffs/edit.html.erb]: @romigos it is duplicated part. Please move to partial _form.html.erb and use it in edit.html.erb and new.html.erb

Comment (romigos) [reply app/views/tariffs/edit.html.erb]: fixed

File: app/views/static_pages/index.html.erb [hunk 12]
```diff
@@ -31,9 +31,12 @@
        <%= render "shared/validation"  %>
         <!-- Admin links-->
     <br>
+    <br>
```
Comment (saviokmua) [inline app/views/static_pages/index.html.erb:34]: @romigos please use css for padding/margin

**Pull request merged**